tests/tests/elasticsearch_tests.cpp

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Please create a separate test executable for the ES tests. I think the standard chain_test suite shouldn't require a working ES server.

Reply:
yes, makes sense, ill take care. thanks.